utilities/getSlugPrefix.ts

```diff
@@ -0,0 +1,8 @@
+export const getSlugPrefix = (): string => {
```

Comment:
New utility function to get the configured slug prefix and ensure no trailing slash.